### DISCUSSION TITLE
Better support for custom fields diff in change log

### DISCUSSION THREAD

**Discussion by empusas** [netbox-community/netbox#13781] [Ideas]
We heavily on the custom_fields feature in netbox. We store supplementing data to most types of objects and also trigger parts of our automation.
I appreciate the functionality of the change log and the diff function. But in regards to custom_fields the diff only seem to list that custom_field dictionary was changed, but not what exactly. That requires an additional step in our automation to compare the fields to figure out which one was changed.
But also for UI users it is difficult to spot the changes.
I would vote for a more granular analyses of the custom fields in the diff of the change log.

**Comment by AndreasSMoe** [comment]
Same here, many custom fields, hard to spot change to custom fields
